example/App.tsx

```diff
@@ -0,0 +1,127 @@
+/**
+ * Sample React Native App
+ * https://github.com/facebook/react-native
+ *
+ * @format
+ */
+
+import React, {useState, useEffect} from 'react';
+import {
+  ScrollView,
+  Button,
+  TextInput,
+  StyleSheet,
+  PermissionsAndroid,
+  Platform,
+  View,
+  Permission,
+} from 'react-native';
+
+import {useJellyfishClient} from '@jellyfish-dev/react-native-client-sdk';
+
+import {Room} from './src/Room';
+import {JELLYFISH_URL} from '@env';
+
+function App(): JSX.Element {
+  const {connect, join, cleanUp} = useJellyfishClient();
+  const [isConnected, setIsConnected] = useState(false);
+  const [peerToken, onChangePeerToken] = React.useState('Peer token');
+
+  useEffect(() => {
+    async function request() {
+      if (Platform.OS === 'ios') {
+        return;
+      }
+      try {
+        const granted = await PermissionsAndroid.requestMultiple([
+          PermissionsAndroid.PERMISSIONS.CAMERA as Permission,
+          PermissionsAndroid.PERMISSIONS.RECORD_AUDIO as Permission,
+        ]);
+        if (
+          granted[PermissionsAndroid.PERMISSIONS.CAMERA as Permission] ===
+            PermissionsAndroid.RESULTS.GRANTED &&
+          granted[PermissionsAndroid.PERMISSIONS.RECORD_AUDIO as Permission] ===
+            PermissionsAndroid.RESULTS.GRANTED
+        ) {
+          console.log('You can use the camera');
+        } else {
+          console.log('Camera permission denied');
+        }
+      } catch (err) {
+        console.warn(err);
+      }
+    }
+
+    request();
+  }, []);
+
+  const connectToRoom = async () => {
+    try {
+      await connect(JELLYFISH_URL, peerToken);
```

Comment:
when connecting to webrtc you have an option to disable audio track (`enableAudioTrack` or something like that)
you can try to use it to solve the double track issue

Reply:
when using this flag webrtc stills sends an audio track, so it does not fix this problem, I'll take a look at this in different PR